components/docs-chef-io/content/automate/centralizing_log.md

```diff
@@ -144,3 +144,16 @@ Restart your splunk forwarder using the following command:
 ```
 
 The above step will let you view the `automate.log` file in your splunk forwarder. Click [here](https://statics.teams.cdn.office.net/evergreen-assets/safelinks/1/atp-safelinks.html) to know about the Splunk Forwarder.
+
+## Frequently Asked Questions
+
+### Is the setting of `redirect_log_file_path` directory where the log files will be stored or the full path of the actual log file?
```

Comment:
This does not look like a FAQ to me. This can be resolved by being more explicit in our document